apps/src/templates/teacherHomepage/CollapsibleSection.jsx

```diff
@@ -69,7 +72,7 @@ const CollapsibleSection = React.createClass({
 
   renderContent() {
     const content = this.props.children;
-    const childItems = _.isArray(this.props.children) ? this.props.children.slice(0, 2) : [this.props.children];
+    const childItems = _.isArray(this.props.children) ? this.props.children : [this.props.children];
```

Comment:
I didnt think of this before, but I bet we can use React.Children (https://facebook.github.io/react/docs/react-api.html#react.children) to more cleanly handle the fact that we have one or more children.